src/safeds/data/tabular/plotting/_table_plotter.py

```diff
@@ -89,13 +96,11 @@ def box_plots(self) -> Image:
         # removes unused ax indices, so there wont be empty plots
         last_filled_ax_index = len(columns) % number_of_columns
         for i in range(last_filled_ax_index, number_of_columns):
-            if number_of_rows == 1:
-                fig.delaxes(axs[i])
-            else:
+            if number_of_rows != 1 and last_filled_ax_index != 0:
                 fig.delaxes(axs[number_of_rows - 1, i])
 
         fig.tight_layout()
-
+        _figure_to_image(fig).to_png_file("C:/Users/patri/Desktop/test.png")
```

Comment:
Absolute paths should not be part of the repo.

To update snapshots, run tests using `pytest --snapshot-update`. `syrupy` automatically takes care of snapshot updates for you. You can also run tests only partially with that flag to speed up the process.